build/config/linux/libc.gni

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2023 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+declare_args() {
+  # The C library used by the compilation host.
+  # Expecting one of: "gnu" (glibc), "uclibc", "musl"
+  host_libc = ""
+
+  # The C library of the compilation target.
+  # Expecting one of: "gnu" (glibc), "uclibc", "musl"
+  target_libc = ""
+}
+
+if (host_libc == "") {
+  # XXX: actually detecting it is complicated
+  host_libc = "gnu"
+}
+if (target_libc == "") {
+  target_libc = host_libc
+}
+
+is_cross_libc = host_libc != target_libc
```

Comment:
Curious where you intend to use this in the engine repo. I suspect that if I saw this identifier out of context, I'd end up grepping the source to see how it's defined, it's short and clear enough that it's probably more readable to just do the host != target comparison at each site in the engine repo where this variable would be used.

Reply:
I've seen a case somewhere where target triplet was getting set specifically for cross-compilation (and a few architectures, where default didn't work), but not when compiling for host. can't find it now, very possibly it was in dart or webrtc, might only be like that in some older versions